libnmstate/schema.py

```diff
@@ -72,6 +72,7 @@ class RouteRule:
     IP_TO = "ip-to"
     PRIORITY = "priority"
     ROUTE_TABLE = "route-table"
+    NEXT_HOP_INTERFACE = "next-hop-address"
```

Comment:
This is not needed. You can find the `next-hop-interface` in Route class. Thanks!